ios/RN/RNCamera.m

```diff
@@ -265,6 +265,24 @@ -(AVCaptureSessionPreset)getDefaultPresetVideo
     return preset;
 }
 
+- (void)lockDevice:(AVCaptureDevice *)device andApplySettings:(void (^)(void))applySettings {
+    NSError *error = nil;
+
+    if(device == nil){
+        return;
+    }
+
+    if (![device lockForConfiguration:&error]) {
+        if (error) {
+            RCTLogError(@"%s: %@", __func__, error);
+        }
+        return;
+    }
+    
+    applySettings();
+    
+    [device unlockForConfiguration];
+}
```

Comment:
I would keep this changes in your existing pull request https://github.com/react-native-community/react-native-camera/pull/2774 

Other Developers will review pull request in the future to build similar functionalities so it would help keeping them separated. Thanks for the awesome work.

Reply:
Thanks again for looking into this. If I get you right, you want me to move the changes I made in RNCamera.m to the other pull request (https://github.com/react-native-community/react-native-camera/pull/2774)?

Reply:
https://github.com/rfischer/react-native-camera/pull/1

Reply:
https://github.com/rfischer/react-native-camera/pull/2